ovs/extensions/services/systemd.py

```diff
@@ -208,7 +208,14 @@ def start_service(name, client):
         except CalledProcessError as cpe:
             output = cpe.output
             Systemd._logger.exception('Start {0} failed, {1}'.format(name, output))
-        return output
+        counter = 0
+        while counter < 10:
+            if Systemd.get_service_status(name=name, client=client)[0] is True:
```

Comment:
Can you verify why exactly that happened? Did the process log something (e.g. a crash)?